### PR TITLE
BUG: wasm_type_from_image_type for VariableLengthVector

### PR DESCRIPTION
**Pull request by thewtex**
And other fixes.

### CODE REVIEW

File: Wrapping/Generators/Python/itk/support/helpers.py [hunk 1]
```diff
@@ -277,38 +267,44 @@ def wasm_type_from_image_type(itkimage):  # noqa: C901
...
     elif component in [i[1] for i in itk.RGBAPixel.items()]:
-        mangle = itk.template(component)[1][0]
+        componentType = python_to_js(itk.template(component)[1][0])
         pixelType = "RGBA"
     elif component in [i[1] for i in itk.RGBPixel.items()]:
-        mangle = itk.template(component)[1][0]
+        componentType = python_to_js(itk.template(component)[1][0])
         pixelType = "RGB"
     elif component in [i[1] for i in itk.SymmetricSecondRankTensor.items()]:
         # SymmetricSecondRankTensor
```
Comment (dzenanz) [inline Wrapping/Generators/Python/itk/support/helpers.py:296]: This comment does not look overly useful.

**Review by dzenanz — APPROVED** [review]

**Comment by PranjalSahu** [comment]
Is there some re-factoring also done along with the change for Variable Length Vector ?

**Review by hjmjohnson — APPROVED** [review]
LGTM.

**Review by PranjalSahu — APPROVED** [review]
LGTM

**Comment by thewtex** [comment]
> Is there some re-factoring also done along with the change for Variable Length Vector ?

It wasn't supported before, which motivated these improvements. Test now present :-)

**Pull request merged**